src/main/scala/org/ergoplatform/settings/ErgoSettingsReader.scala

```diff
@@ -21,14 +22,19 @@ import scala.util.Try
 object ErgoSettingsReader extends ScorexLogging
   with PowSchemeReaders
   with NodeConfigurationReaders
-  with SettingsReaders {
+  with SettingsReaders
+  with FileUtils {
 
   def read(args: Args = Args.empty): ErgoSettings = {
     fromConfig(readConfig(args), args.networkTypeOpt)
   }
 
   def fromConfig(config: Config, desiredNetworkTypeOpt: Option[NetworkType] = None): ErgoSettings = {
-    val directory = config.as[String](s"$configPath.directory")
+    val directory =
```

Comment:
Better to avoid such hacks in settings reader, also, for integration tests it would be needed to work with directory provided in a config

Reply:
done